azure/basic-aks-cluster/README.md

```diff
@@ -10,25 +12,42 @@ If you're new to XebiaLabs blueprints, check out:
 * [Get started with blueprints](https://docs.xebialabs.com/xl-platform/concept/get-started-with-blueprints.html)
 * [Get started with XL JetPack](https://docs.xebialabs.com/xl-platform/concept/get-started-with-xl-jetpack.html)
 
-## Prerequisites
+## Usage
+
+To use this blueprint, run `xl blueprint` in an empty directory and select:
+
+```plain
+azure/basic-aks
+```
+
+## Tools and technologies
+<!--Dev create this section for consistency with other blueprint readmes (see https://github.com/xebialabs/blueprints/tree/master/gcp/microservice-ecommerce for example)-->
+
+## Minimum required versions
 
-### Authenticate to Azure
+This blueprint version requires at least the below versions of the specified tools to work properly.
+
+- XL Deploy: Version 8.6.1
+- XL CLI: Version 8.6
```

Comment:
I recommend consistently using all three parts of the version number. I'm guessing CLI 8.6 probably means CLI 8.6.0, but it's better to spell it out explicitly than to just imply it.

Reply:
makes sense. will seek and destroy

Reply:
done